pkg/webui/console/components/events/previews/uplink-message.js

```diff
@@ -49,7 +49,7 @@ const UplinkMessagePreview = React.memo(({ event }) => {
     <DescriptionList>
       <DescriptionList.Byte title={messages.devAddr} data={devAddr} />
       <DescriptionList.Item title={messages.fPort} data={fPort} />
-      <DescriptionList.Item title={messages.fCnt} data={fCnt} />
+      <DescriptionList.Item title={messages.fCnt} data={fCnt} highlight />
```

Comment:
```suggestion
      <DescriptionList.Item title={messages.fCnt} data={fCnt} highlight />
      <DescriptionList.Item title={messages.fPort} data={fPort} />
```

Let's also change the order here, which I think is more in line with the order of importance.